src/api/app/components/notification_action_bar_component.rb

```diff
@@ -0,0 +1,36 @@
+# frozen_string_literal: true
+
+class NotificationActionBarComponent < ApplicationComponent
```

Comment:
`NotificationActionBar` is quite a generic name. We could name this component `MarkAsReadButton`. Splitting up this component in multiple components like `MarkAsUnreadButton` and `MarkAllAsReadButton` would be maybe too much abstraction. So having a single component is good enough, at least for now.

Reply:
`NotificationActionBar` has three things. `Select All`, `Mark as read`, and `Mark all as read`. Then the same actions for unread operation. Naming it `MarkAsReadButton` would be a little confusing because we are leaving out the unread operations. We only have one notifications page and one notification action bar. `NotificationActionBar` still seems a better choice. What do you think @dmarcoux ? Should we still rename it?

Reply:
Oh, true I totally overlooked that this component also has the `Select All` checkbox.

Renaming this component isn't needed then.